src/hdf5/spire_2ioda.py

```diff
@@ -219,6 +222,10 @@ def adjust_dateTime(obs_DF, dat_ref):
     obs_DF['dateTime'] = obs_DF['dateTime']+int(dat_ref)
     return obs_DF
 
+def adjust_longitude(obs_DF):
+    obs_DF['longitude'] = (obs_DF['longitude']+int(180)) % 360 - int(180)
```

Comment:
Thanks, @smaticka. Not clear why `int()`?

Reply:
it might not matter. It was mostly because an `int` takes less storage space (bytes) than a `float` and if it's being added and then subtracted, I believe it reduces the amount of computational error (though probably very minor).

Reply:
@smaticka it's redundant. int(180) is the same as 180. it just looks weird.

Reply:
if ObsDF is a float see my suggestion.